aicoe/sesheta/chatbot.py

```diff
@@ -52,6 +52,9 @@
  "sub-mod",
  "xtuchyna",
 ]
+_CHATBOT = ChatBot("Sesheta")
```

Comment:
It mighe be good idea to have the bot in read-only mode - [see the docs](https://chatterbot.readthedocs.io/en/stable/examples.html#read-only-mode):

```suggestion
_CHATBOT = ChatBot("Sesheta", read_only=True)
```

Reply:
We don't want it to learn? :stuck_out_tongue:

Reply:
To my understanding the flag turns on "contextual" conversation which might be not the right thing for long-living bot in a channel.